src/scripts/farming/Farming.ts

```diff
@@ -271,94 +271,94 @@ class Farming implements Feature {
             [
                 'This Berry contains a substance that generates heat. It can even heat up a chilly heart.',
                 'Growing these Berries will promote Egg growth.',
-            ], new Aura(AuraType.Egg, [1.01, 1.02, 1.03]), ['Riolu', 'Scraggy', 'Crabrawler', 'Oricorio (Baile)']);
+            ], new Aura(AuraType.Egg, [1.01, 1.02, 1.03]), ['Mankey', 'Makuhita', 'Riolu', 'Scraggy', 'Crabrawler', 'Oricorio (Baile)']);
         this.berryData[BerryType.Kebia]     = new Berry(BerryType.Kebia,    [100, 200, 400, 600, 86400],
             1, 1, 50, 1,
             [0, 15, 0, 0, 10], BerryColor.Green,
             [
                 'This Berry is a brilliant green on the outside. Inside, it is packed with a dry-flavored, black-colored flesh.',
                 'It has a tendency to overtake nearby plants.',
                 'Due to its poisonous nature, it increases the chances of mutations near it.',
-            ], new Aura(AuraType.Mutation, [1.2, 1.4, 1.6]), ['Gulpin','Salandit', 'Toxel']);
+            ], new Aura(AuraType.Mutation, [1.2, 1.4, 1.6]), ['Grimer', 'Spinarak', 'Gulpin', 'Skorupi', 'Salandit', 'Toxel']);
         this.berryData[BerryType.Shuca]     = new Berry(BerryType.Shuca,    [7200, 16200, 32400, 39600, 79200],
             28, 1, 1700, 15,
             [10, 0, 15, 0, 0], BerryColor.Yellow,
             [
                 'The sweetness-laden pulp has just the hint of a hard-edged and fragrant bite to it.',
                 'Growing these Berries will soften the ground around it, increasing the chances of replanting.',
-            ], new Aura(AuraType.Replant, [1.01, 1.02, 1.03]), ['Larvitar', 'Flabébé (Yellow)', 'Oricorio (Pom-Pom)']);
+            ], new Aura(AuraType.Replant, [1.01, 1.02, 1.03]), ['Sandshrew', 'Larvitar', 'Numel', 'Hippopotas', 'Flabébé (Yellow)', 'Oricorio (Pom-Pom)']);
         this.berryData[BerryType.Coba]      = new Berry(BerryType.Coba,     [9000, 12600, 16200, 19800, 39600],
             29, 0.05, 1800, 15,
             [0, 10, 0, 15, 0], BerryColor.Blue,
             ['This Berry is said to be a new kind that is a cross of two Berries brought together by winds from far away.'],
-            undefined, ['Tropius', 'Flabébé (Blue)']);
+            undefined, ['Farfetch\'d', 'Murkrow', 'Tropius', 'Chatot', 'Flabébé (Blue)']);
         this.berryData[BerryType.Payapa]    = new Berry(BerryType.Payapa,   [4680, 11880, 23400, 34200, 68400],
             30, 0.05, 1900, 15,
             [0, 0, 10, 0, 15], BerryColor.Purple,
             [
                 'This Berry is said to sense human emotions for the way it swells roundly when a person approaches.',
                 'The same behavior affects nearby plants, causing additional mutations.',
-            ], new Aura(AuraType.Mutation, [1.1, 1.2, 1.3]), ['Natu', 'Ralts', 'Oricorio (Sensu)']);
+            ], new Aura(AuraType.Mutation, [1.1, 1.2, 1.3]), ['Mr. Mime', 'Natu', 'Ralts', 'Chingling', 'Oricorio (Sensu)']);
```

Comment:
Gives another option to get QP pokemon if someone were willing to mutate that much of the farm in kanto instead of the small amount of QP